### ISSUE TITLE
How to use it? What am I doing wrong?

### ISSUE THREAD

**Issue by SomaKohsei** [LAVI-USP/pyDBT#3]
Hello,

I have used this software. When using it with other people's data, valid images were obtained (https://github.com/LAVI-USP/DBT-Reconstruction/issues/5).

I manually captured images using an X-ray imaging device and an imaging plate (IP). I adjusted the incidence point to be at the center of the IP and captured images at various angles (2-degree intervals).

The resulting images were placed in a dicom folder. However, all the images obtained have pixel values ​​of 0.

Could you please tell me what I am doing wrong in my approach?
Is the file format of the image not good enough? (16-bit? signed?)?

I have uploaded the necessary files to the following URL.
https://www.mediafire.com/file/bhuph8wt8fghnc2/pyDBT2.zip/file

![pic](https://user-images.githubusercontent.com/35340150/236854886-6900b7b1-da02-4239-aaeb-209b84a22eda.JPG)
![pic2](https://user-images.githubusercontent.com/35340150/236855101-a3763daf-88b0-42a0-823c-276f0292c20e.JPG)

**Comment by rodrigovimieiro** [comment]
pyDBT was meant for Breast Tomosyhthesis. It uses half-cone beam, which I guess it is not your case.

You may want to use [Tigre](https://github.com/CERN/TIGRE) for your geometry. It is a CT toolbox which is very flexible and you perhaps can adjust it for your geometry.

**Comment by SomaKohsei** [comment]
Thank you for your response.
Indeed, as you pointed out, my geometry is different from breast tomosynthesis.
Would it be possible to make it a half-cone beam by doing as shown in the figure below?
Even if it is not a half-cone beam, it seems strange that it does not appear at all in the reconstructed image.
![pic3](https://user-images.githubusercontent.com/35340150/236969635-73182dfe-878a-49cc-bd5e-4e8bca497e08.JPG)

**Comment by rodrigovimieiro** [comment]
Yes, you can do it, but I guess it would be just easier to use the full cone-beam toolbox. 

I even adapted TIGRE to work with [DBT images](https://github.com/CERN/TIGRE/blob/master/MATLAB/Demos/d19_DBTexample.m). You might get some insights there on how to change the geometry to your needs.

You can also adapt pyDBT to work with a full cone beam. I just don't really know how much work needs to be done.

Let me know if it is clear to you.

**Comment by SomaKohsei** [comment]
I apologize for the delayed response. I didn't have Matlab, so I purchased it. I'm currently struggling with the installation of TIGRE since I have no prior experience using Matlab. I will report back once I obtain the images.

**Comment by SomaKohsei** [comment]
Hello,
I purchased MATLAB and tried using TIGRE's Demo19. It might not be appropriate to mention TIGRE-related matters here, so please forgive me.
When I look at the resulting image, I can faintly see bone structures in the background. (Is it a double reflection?) 

Since I have little experience in viewing tomosynthesis images obtained from commercial devices, I'm not sure if the results I obtained are correct. I would appreciate your opinion.

thank you
Soma

![output](https://github.com/LAVI-USP/pyDBT/assets/35340150/8e8cac43-a9fc-4772-aa80-5af98dd9931a)

**Comment by rodrigovimieiro** [comment]
Hello,

It is hard to tell. I can see things coming into focus and then going out of focus. We can see that type of behavior in DBT images, so I would say it is somehow close to the real one.

**Comment by sokusekigit** [comment]
Hello, 

Thank you for the comment.
I am satisfied with the result. I look forward to capturing various subjects from now on.
Thank you for the amazing software.

**Comment by SomaKohsei** [comment]
> Hello,
> 
> It is hard to tell. I can see things coming into focus and then going out of focus. We can see that type of behavior in DBT images, so I would say it is somehow close to the real one.

Thank you for the comment.
I am satisfied with the result. I look forward to capturing various subjects from now on.
Thank you for the amazing software.
